pkg/server/logging.go

```diff
@@ -0,0 +1,54 @@
+package server
+
+import (
+	"github.com/gorilla/handlers"
+	"go.uber.org/zap"
+	"io"
+	"net"
+)
+
+func httpLogFormatter(logger *zap.Logger) func(io.Writer, handlers.LogFormatterParams) {
+	if logger == nil {
+		logger = zap.NewNop()
+	}
+
+	return func(_ io.Writer, params handlers.LogFormatterParams) {
+		var req = params.Request
+
+		username := "-"
```

Comment:
Feels like user could be removed from the output

Reply:
You mean remove the entire `username` field or only add username field when it's set or just let it be an empty string?

Reply:
Remove username entirely and in the output on line 45

Reply:
Removed in https://github.com/jhaals/yopass/pull/1130/commits/8c76a4a0faee824f07058f370c4be492cfa74d32